odl/contrib/datasets/images/cambridge.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Images provided by the University of Cambridge."""
+
+import numpy as np
+from odl.contrib.datasets.util import get_data
+
+
+__all__ = ('brain_phantom', 'resolution_phantom', 'cms', 'cms_gray', 'rings',
+           'blurring_kernel')
+
+
+DATA_SUBSET = 'images'
+URL_CAM = 'http://store.maths.cam.ac.uk/DAMTP/me404/data_sets/'
+
+
+def brain_phantom():
```

Comment:
Would it be possible to add a flag to this to return this phantom in different forms, e.g.

- `'materials'` Materials by index (e.g. 0, 1, 2, 3).
- `'PET'` The pet phantom you have right now I guess
- `'density'`/`'CT'` Corresponding HU/density values
- `'MRI'` Fake proton densities

That would make this much more applicable!

Reply:
Well, I have the code somewhere in MATLAB where you can define intensity levels for meaningful regions. If you are interested I can share this with you but I think I won't be interested in making this available to ODL myself.

Reply:
The main reason why I am not interested in this any more is because it is intrinsically 2D. To have a meaningful 3D version of this would be good.